tests/components/monoprice/test_media_player.py

```diff
@@ -373,15 +464,13 @@ def test_is_volume_muted(self):
 
     def test_supported_features(self):
         """Test supported features property."""
-        assert (
-            SUPPORT_VOLUME_MUTE
-            | SUPPORT_VOLUME_SET
-            | SUPPORT_VOLUME_STEP
-            | SUPPORT_TURN_ON
-            | SUPPORT_TURN_OFF
-            | SUPPORT_SELECT_SOURCE
-            == self.media_player.supported_features
-        )
+        assert (SUPPORT_VOLUME_MUTE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4570914)